packages/macro/src/index.ts

```diff
@@ -149,3 +149,29 @@ function getMacroType(tagName: string): string {
 }
 
 export default createMacro(macro)
+
+const macros = [
+  'defineMessage',
+  'arg',
+  't',
+  'plural',
+  'select',
+  'selectOrdinal',
+
+  // jsx
+  'Trans',
+  'Plural',
+  'Select',
+  'SelectOrdinal',
+];
```

Comment:
I don't think it is a good idea to have two identical lists of tag names (the other is in the `getMacroType` function). Please merge these two concepts into a dictionary, update the `forEach` below, remove the `getMacroType` function and update getting the macro type on the line 43.